requirements.txt

```diff
@@ -13,14 +14,21 @@ django-oauth-toolkit==0.12.0
 django-storages==1.5.1
 djangorestframework==3.5.3
 docopt==0.6.2
+enum-compat==0.0.2
+flake8==3.3.0
+flake8-import-order==0.12
+flake8-strict==0.1.4
+flake8-tidy-imports==1.0.5
 freezegun==0.3.8
 isort==4.2.5
 lazy-object-proxy==1.2.2
-mccabe==0.5.2
+Markdown==2.6.8
```

Comment:
You are right. I am not sure, how to work with this this requirements file, because it is out of sync with `pip freeze` at the moment.
In my project I am using two requirements files - `requirements.txt` with source dependencies and then `requirements.freeze.txt` which is generated with `pip freeze` to ensure, that it runs with same version of requirements that it has been tested with.

I will fix this PR to contain only needed dependencies now.